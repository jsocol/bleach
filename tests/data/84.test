<div id="13"><x repeat="template" repeat-start="999999">0<y repeat="template" repeat-start="999999">1</y></x>//["'`-->]]>]</div>
