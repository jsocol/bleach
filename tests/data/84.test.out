<div id="13">01//["'`--&gt;]]&gt;]</div>
